src/test/auto-completer_test.ts

```diff
@@ -474,6 +474,79 @@ suite('AutoCompleter', () => {
               'polymer/element-with-databinding.html', internalPropUsePosition),
           databindingCompletions);
     });
+
+
+    test(`autocomplete css custom properties`, async() => {
```

Comment:
nit: can you separate this in 2 tests - 1 for declaration and 1 for completion?

Reply:
Sure, done